README.md

```diff
@@ -1,18 +1,40 @@
-# DRAFT ONLY : UNTIL KEP IS MERGED 
-  https://github.com/kubernetes/enhancements/pull/1383
+COSI repos:
+
+|Repo | Images | 
+|----------------|-----------|
+| [Spec](https://github.com/kubernetes-sigs/container-object-storage-interface-spec) | N/A  |
+| [API](https://github.com/kubernetes-sigs/container-object-storage-interface-api) | N/A |
+| [Controller](https://github.com/kubernetes-sigs/container-object-storage-interface-controller) | [cosi-controller](https://quay.io/repository/containerobjectstorage/objectstorage-controller?tab=tags) |
+| [Provisioner Sidecar](https://github.com/kubernetes-sigs/container-object-storage-interface-provisioner-sidecar) | cosi-provisioner TODO |
+| [CSI Adapter](https://github.com/kubernetes-sigs/container-object-storage-interface-csi-adapter) | cosi-node-adapter TODO |
+\
```

Comment:
Please fix this.

Reply:
There are no images created on quay.io to my knowledge for the provisioner-sidecar and the CSI adapter. Were you referring to something else?

Reply:
oh I see, will fix